dev-docs/source/Standards/ReleaseNotesGuide.rst

```diff
@@ -51,8 +51,7 @@ another sub-heading (e.g. Powder Diffraction, Algorithms, MSlice), you need to d
 
 - Add a new directory into the correct part of the filing system. E.g. to add Algorithms to Workbench create the directory ``/Workbench/Algorithm``. The directory name should not contain any spaces.
 - Add ``New_features`` and ``Bugfixes`` directories within your new directory. The automated script only works with these directories.
-- Add a blank .rst file to each of the ``New_features`` and ``Bugfixes`` directories that has the same name as the directory - this is where the automated script amalgamates the separate release notes.
-- Update the top level release note file with your new heading and sub-headings. For each subheading you need to add an include statement with a link to each new blank .rst file to ensure the notes will be visible for users. For this example it may look something like this
+- Update the top level release note file with your new heading and sub-headings. For each subheading you need to add an ``amalgamate`` statement with a link to each new directory to ensure the notes will be visible for users. The link to the new directory must start with `/` to enable the script to work. For this example it may look something like this
```

Comment:
We should check for this leading `/` and emit some sort of exception if it's missing, as I can see it being a common mistake

Reply:
I've incoporated this into a ned method which will generate the path